docs/collecting-data/collecting-from-own-applications/mobile-trackers/anonymous-tracking/index.md

```diff
@@ -17,15 +17,11 @@ This feature is available since v4.
 
 :::
 
-Anonymous tracking is a tracker feature that enables anonymising various user and session identifiers to support user privacy in case consent for tracking the identifiers is not given.
-
-The affected user and session identifiers are stored in two context entities: [Session](../tracking-events/index.md#session) and [Platform context](../tracking-events/index.md#platform). The Session context entity contains user and session identifiers, while the Platform context entity contains user identifiers. Concretely, the following user and session identifiers can be anonymised:
-
-1. Client-side user identifiers: the `userId` in Session context entity and the IDFA identifiers (`appleIdfa`, and `appleIdfv`) in the Platform context entity.
-2. Client-side session identifiers: `sessionId` in Session context.
-3. Server-side user identifiers: `network_userid` and `user_ipaddress` event properties.
+```mdx-code-block
+import Block8155 from "@site/docs/reusable/anonymous-tracking-mobile/_index.md"
```

Comment:
You don’t need to give it a silly name like `Block8155`, I only had to do that in other pages because it was an automated import from Wordpress and I used Wordpress IDs to make sure multiple blocks on the same page don’t clash :)

Reply:
True, fixed.

Reply:
What about here? :)